file/test/file_endpoint_test.exs

```diff
@@ -0,0 +1,228 @@
+defmodule Membrane.RTC.FileEndpointTest do
+  use ExUnit.Case
+
+  import Membrane.ChildrenSpec
+
+  alias Membrane.RTC.Engine
+  alias Membrane.RTC.Engine.Endpoint.File, as: SourceEndpoint
```

Comment:
I would use the original name

Reply:
If we do that, we wouldn't be able to use functions from `Elixir.File` module, e.g: `File.exists?` or `File.read!`

Reply:
Let's then use `Endpoint.File`